templates/zerver/emails/missed_message.subject.txt

```diff
@@ -1,10 +1,10 @@
 {% if show_message_content %}
-    {% if group_pm %} Group PMs with {{ huddle_display_name }}
-    {% elif private_message %} PMs with {{ sender_str }}
+    {% if group_pm %} {% trans %}Group PMs with{% endtrans %} {{ huddle_display_name }}
+    {% elif private_message %} {% trans %}PMs with{% endtrans %} {{ sender_str }}
```

Comment:
Same here as well.

Reply:
Can't do that here due to the presence of conditional statements. I tried wrapping the multiple lines in a single `trans` tag, but got an error saying the previous statement. So had to do separate tags for each conditional.

Reply:
I meant to keep the ` {{ sender_str }}` string inside the trans tags.